### DISCUSSION TITLE
Sources for Job Opportunities

### DISCUSSION THREAD

**Discussion by Meharban-Singh** [UWCodeForce/student-net#6] [Ideas]
- In the job opportunities page, what sources should we add. 
- Additionally, does that resource has an API, or does it need scraping? 
- What filters does the source provide to filter for student jobs? 

Anyone who has a resource can comment here with answer to the questions above for that source. This would give us a final list of sources to add.

**Comment by Raghuboi** [comment]
Indeed 
- API: unclear, but I was looking at this popular repository for an indeed-job-scraper https://github.com/israel-dryer/Indeed-Job-Scraper
- Filters: Job Title, Company, Location, Post Date, Salary, Job Type (Part time/Full Time)
- Additional Data: Job Description, URL

Update: 

They have an API (https://developer.indeed.com/) along with a javascript client library for it (https://github.com/indeedlabs/indeed-js) although the most recent one is for python (https://github.com/indeedlabs/indeed-python)

**Comment by Raghuboi** [comment]
Glassdoor
- API: https://www.glassdoor.com/developer/index.htm **but you need to contact them to get a key**
- Filters: Job Title, Job Function, Industry, Company, State/City, Salary (avg base salary estimate)
- Additional Data: Job Description, Salary Range (upper bound & lower bound), URL
- More Additional Data: Company Rating, Company Size, Company Revenue/Type (Public/Private)/Sector/Founded (all optional fields)
